src/app/applicationComponents/Main/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import NewsList from '../../reusableComponents/NewsList';
+import newsAppConfig from '../../configFiles/newsAppConfig';
+
+class Main extends React.Component {
```

Comment:
What is "main" component? for what does it stand for?
Please name it more specific to it's functionality

Reply:
Renamed to RouteContentContainer.